### ISSUE TITLE
Recover already ended estimation

### ISSUE THREAD

**Issue by tomaszzelezny** [cschleiden/azure-boards-estimate#38]
Is there a way to bring back already closed estimations?

**Comment by cschleiden** [comment]
No, they are deleted. The actual values are stored in the work items. Is there anything you'd need from deleted sessions?

**Comment by tomaszzelezny** [comment]
It seems like people who are not familiar with this plugin has ended this session by mistake. 

I have idea for some improvements here:
1.  Sessions should be not visible in the other project
2. Instead of deleting session data at the end filter it out on the default view

**Comment by mjversagility** [comment]
I understand they are deleted, what if it was an async vote, the values haven't been revealed and acceptexd?
